generators/client/files-react.js

```diff
@@ -283,56 +284,52 @@ const files = {
     ],
     clientTestFw: [
         {
-            path: CLIENT_TEST_SRC_DIR,
+            path: REACT_DIR,
             templates: [
-                'jest.conf.js',
-                'spec/icons-mock.ts',
```

Comment:
Without code from `icons-mock.ts` console is full of errors "Could not find icon".
Maybe instead of deleting this file move this to `src/main/webapp/app/setup-tests.ts` and still use on Jest config?

Reply:
ok